index.js

```diff
@@ -8,7 +8,8 @@ var cachedDependencies = [];
 
 var defaultOptions = {
   cache: false,
-  yes: true
+  yes: true,
+  warn: true
```

Comment:
As @eeue56 pointed out to me, https://github.com/elm-lang/elm-make/issues/27 means turning on `warn` by default won't give you all the warnings every time, only when files change and recompile...that seems like it would be surprising behavior, so I'd say let's revisit this default if the status https://github.com/elm-lang/elm-make/issues/27 changes.